src/software/ai/hl/stp/play/penalty_kick_enemy/penalty_kick_enemy_play_fsm.cpp

```diff
@@ -0,0 +1,48 @@
+#include "software/ai/hl/stp/play/penalty_kick_enemy/penalty_kick_enemy_play_fsm.h"
+
+PenaltyKickEnemyPlayFSM::PenaltyKickEnemyPlayFSM(TbotsProto::AiConfig ai_config)
+    : ai_config(ai_config), move_tactics(std::vector<std::shared_ptr<MoveTactic>>())
+{
+}
+
+void PenaltyKickEnemyPlayFSM::setupPosition(const Update &event)
+{
+    PriorityTacticVector tactics_to_run = {{}};
+
+    // Adjust number of tactics based on the number of robots available
+    unsigned int num_tactics = event.common.num_tactics;
+    if (num_tactics != move_tactics.size())
+    {
+        move_tactics = std::vector<std::shared_ptr<MoveTactic>>(num_tactics);
+        std::generate(move_tactics.begin(), move_tactics.end(),
+                      []() { return std::make_shared<MoveTactic>(); });
+    }
+
+    // Move all robots behind the penalty mark
+    for (unsigned int i = 0; i < move_tactics.size(); i++)
+    {
+        double y_offset = 4 * ((double)i - ((double)move_tactics.size() - 1) / 2.0) *
```

Comment:
nit: can we increase the offset just to give the robots a bit more space to motion plan around each other?
```suggestion
        double y_offset = 8 * ((double)i - ((double)move_tactics.size() - 1) / 2.0) *
```

Reply:
done, also adjusted offsets in `penalty_kick_play_fsm.cpp`